build.gradle

```diff
@@ -27,7 +29,7 @@ repositories {
 dependencies {
     testCompile 'cglib:cglib-nodep:3.2.9'
     testCompile 'org.apache.commons:commons-io:1.3.2'
-    testCompile('org.spockframework:spock-core:1.2-groovy-2.4')
+    testCompile('org.spockframework:spock-core:1.2-groovy-2.5')
```

Comment:
Any specific reason not to update to `1.3-groovy-2.5`?

Reply:
Seems to work fine locally, pushed the change.